util/serial/Cargo.toml

```
@@ -9,9 +9,9 @@ edition = "2018"
 # because rmp-ser../../alloc doesn't build against ser../../std,
 # so if anything else in your build plan will activate ser../../std, then mcseri../../std is
 # required.
-std = [ "serde/std", "serde_cbor/std" ]
+std = [ "serde/std" ]
```

IIUC we want to pull in that package's `std` feature
```suggestion
std = [ "serde/std", "minicbor-ser/std" ]
```